cat-test/lib/cat.dart

```diff
@@ -0,0 +1,10 @@
+import 'cat_cafe.dart';
+
+class Cat {
+  final service;
```

Comment:
`final CatCafe service`. Include types for any instance variables that don't have initializers from which the type can be inferred. Otherwise, inference and dartdoc don't work properly.